src/Manager.php

```diff
@@ -24,18 +25,23 @@ final class Manager implements AccessCheckerInterface
 
     public function __construct(StorageInterface $storage, RuleFactoryInterface $ruleFactory)
     {
-        $this->storage = $storage;
+        $this->storage = $this->assignmentStorage = $storage;
```

Comment:
I'm not sure this is following the Yii code style (should be on two separate lines)